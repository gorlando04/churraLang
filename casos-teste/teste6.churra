amigo "A" contribui 200
amigo "B" contribui 200
amigo "C" contribui 200
item carne "picanha" preco_kg 60 quantidade_kg 2
item carne "fraldinha" preco_kg 50 quantidade_kg 2
item comida preco_unit 20 quantidade 2
rateio igual