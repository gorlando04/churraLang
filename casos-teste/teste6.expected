Total do churrasco: R$260.00
A deve pagar: R$86.67
B deve pagar: R$86.67
C deve pagar: R$86.67